### PR TITLE
feat(config): Update the config to support sky in the style json. BM-1052

### PR DESCRIPTION
**Pull request by Wentao-Kuang**
### Motivation

Update the config style parser and loader to include sky in the stylejson.

### Modifications

Add sky type into stylejson

### Verification

image
image

### CODE REVIEW

File: packages/config/src/config/vector.style.ts [hunk 1]
```diff
@@ -40,6 +40,24 @@ export interface Layer {
   'source-layer'?: string;
 }
 
+/** Sky MapLibre Style Spec, all valuables support interpolate expressions, as unknown format */
+export interface Sky {
+  /** The base color for the sky. Optional Defaults to #88C6FC */
+  'sky-color'?: unknown;
```
Comment (blacha) [inline packages/config/src/config/vector.style.ts:46]: since we know what these are meant to be could we type them properly? `string` `number` etc?

Comment (Wentao-Kuang) [reply packages/config/src/config/vector.style.ts:46]: These all support interpolate expressions, we can use `string | unknown` or just `unknown`

Comment (blacha) [reply packages/config/src/config/vector.style.ts:46]: `string | unknown[]` ?

Comment (Wentao-Kuang) [reply packages/config/src/config/vector.style.ts]: Updated

**Review by blacha — APPROVED** [review]

**Pull request merged**